stdlib/asyncio/events.pyi

```diff
@@ -55,16 +55,22 @@ class TimerHandle(Handle):
     if sys.version_info >= (3, 7):
         def when(self) -> float: ...
 
-class AbstractServer:
+class AbstractServer(metaclass=ABCMeta):
```

Comment:
We shouldn't lie about the metaclass in typeshed. It's not necessary. Mypy will error if you try to instantiate a class that doesn't override methods decorated with `@abstractmethod`, even if the base class doesn't have `ABCMeta` as its metaclass.